docs/swagger_v3/oracles.spec.yaml

```diff
@@ -534,7 +494,12 @@ paths:
                       data:
                         type: array
                         items:
-                          $ref: '#/components/schemas/OracleQuery'
+                          allOf:
+                            - $ref: '#/components/schemas/OracleQuery'
+                            - type: object
+                              properties:
```

Comment:
`example:` omitted because it is actually the same as the example in `OracleResponse`